pydockerize.py

```diff
@@ -84,14 +108,9 @@ def get_cmd_from_procfile(procfile):
     return lines[0].split(':')[1].strip()
 
 
-def write_dockerfile(base_image, requirements_file, cmd, entrypoint):
-    print('write_dockerfile: base_image = %r' % base_image)
-    print('write_dockerfile: requirements_file = %r' % requirements_file)
-    print('write_dockerfile: cmd = %r' % cmd)
-    print('write_dockerfile: entrypoint = %r' % entrypoint)
-
-    filename = 'Dockerfile-' + base_image
-    print('write_dockerfile: Writing %s' % filename)
+def write_dockerfile(base_image, requirements_file, filename, cmd, entrypoint):
+    click.echo('write_dockerfile: base_image = %r' % base_image)
```

Comment:
Only one letter difference between write_dockerfiles and write_dockerfile could be confusing in the output, maybe?